docs-gen/content/docs/getting-started/overview.md

```diff
@@ -3,54 +3,23 @@ title: "Overview"
 weight: 1
 ---
 
-The Kubeflow Pipelines Operator provides a declarative API for managing and running ML pipelines with Resource Definitions on multiple providers.
+The Kubeflow Pipelines Operator (KFP Operator) provides a declarative API for managing and running ML pipelines with Resource Definitions on multiple providers.
 A provider is a runtime environment for managing and executing ML pipelines and related resources.
 
-## Compatibility
+### Why KFP Operator
 
-The operator currently supports
-- TFX Pipelines with Python 3.7 and 3.9 - pipelines created using the KFP DSL are not supported yet
-- KFP standalone (a full KFP installation is not supported yet) and Vertex AI
+The idea behind the project came about because as engineers we wanted to apply our engineering best practices behind the 
+process of deploying a trained model to be served in a production environment. We wanted to get away from the mentality of 
+"I just need this trained model in Production" or the "If it's 10 lines of code I'd rather copy and paste it".
 
-## TFX Pipelines and Components
+Data scientists produce the artifact of the "model training pipeline code" (which is fully promotable through the environments)
+and then the deployment configuration for the pipeline etc. can be applied in the different environments, giving the required
+training in each environment on the path to production. Linked with the serving component then gives you a fully testable path to production for the model.
```

Comment:
“Data scientists" can mean a lot of things in the industry and I don't think we should use that term here. Also wanted to clarify how simple resources are all that's needed - "deployment configuration for the pipeline etc" might be confusing
```suggestion
By configuring simple Kubernetes resources, machine learning practitioners can run their desired training pipelines in each environment on the path to production in a repeatable, testable and scalable way. When linked with serving components, this provides a fully testable path to production for machine learning systems.
```